### PR TITLE
Mobile menu - first draft

### PR DESCRIPTION
**Pull request by lustoykov**
First iteration of mobile menu - dropdown for tracks is missing #104

### CODE REVIEW

File: src/components/shared/nav/index.jsx [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+        .getElementsByTagName('body')[0]
+        .classList.add('u-disable-scroll');
+    } else {
+      document
+        .getElementsByTagName('body')[0]
+        .classList.remove('u-disable-scroll');
+    }
+  }, [isOpen]);
+
+  console.log({ isOpen, setIsOpen });
```
Comment (lustoykov) [inline src/components/shared/nav/index.jsx]: :D

File: src/components/shared/nav/index.jsx [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+          style={{
+            position: 'fixed',
+            top: '86px',
+            right: 0,
+            left: 0,
+            bottom: 0,
+            backgroundColor: 'rgba(0,0,0, 0.56)',
+          }}
+        ></div>
+      )}
```
Comment (mstockerl) [inline src/components/shared/nav/index.jsx]: Can you explain (or add a comment) what this thing does?

File: src/components/shared/nav/index.jsx [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+            className="nav__menu__item nav__menu__item--mobile"
+            onClick={() => setIsOpen(false)}
+          >
+            <Link to="/apply"> Apply </Link>
+          </li>
+          <li
+            className="nav__menu__item nav__menu__item--mobile"
+            onClick={() => setIsOpen(false)}
+          >
+            <Link to="/faq"> FAQ </Link>
```
Comment (mstockerl) [inline src/components/shared/nav/index.jsx]: Is it really necessary to have the menu twice? 
If yes, I understand, why my approaches failed :D

File: src/components/shared/nav/index.jsx (hunk 2 above)
Comment (lustoykov) [reply src/components/shared/nav/index.jsx]: lol yep 👍

File: src/components/shared/nav/index.jsx (hunk 3 above)
Comment (lustoykov) [reply src/components/shared/nav/index.jsx]: I think some css pros can do both menus with the same markup but would be too much effort for me personally to figure it out

**Pull request merged**